src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -143,7 +143,7 @@
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
     
     <!-- These symbols will be returned from the GetSemanticInfo API if it examines this bound node. -->
-    <Field Name="Symbols" Type="ImmutableArray&lt;Symbol&gt;"/>
+    <Field Name="Symbols" Type="ImmutableArray&lt;Symbol?&gt;" SkipInNullabilityRewriter="true"/>
```

Comment:
>Symbol? [](start = 50, length = 7)

Do we have cases where the array contains `null` values?

Reply:
Yes, the tests that failed on my last commit were because of `null`s in this array. [https://dev.azure.com/dnceng/public/_build/results?buildId=358008&view=ms.vss-test-web.build-test-results-tab](https://dev.azure.com/dnceng/public/_build/results?buildId=358008&view=ms.vss-test-web.build-test-results-tab)

---
In reply to: [325904635](https://github.com/dotnet/roslyn/pull/38736#discussion_r325904635) [](ancestors = 325904635)

Reply:
That feels like a bug in the code path that created the `BoundBadExpression`. Consider leaving this as `Symbol!` to try to catch those cases in the future.

---
In reply to: [325911724](https://github.com/dotnet/roslyn/pull/38736#discussion_r325911724) [](ancestors = 325911724,325904635)

Reply:
I've opened [https://github.com/dotnet/roslyn/issues/38765](https://github.com/dotnet/roslyn/issues/38765) to track. For now, I'm going to leave the annotation as is, because it's correct as written.

---
In reply to: [325917220](https://github.com/dotnet/roslyn/pull/38736#discussion_r325917220) [](ancestors = 325917220,325911724,325904635)